### ISSUE TITLE
Log call without tags shows -

### ISSUE THREAD

**Issue by 5an1ty** [felixheck/laabr#13]
Log with tags:
```
2018-07-23T12:12:41.407Z [info] ["INIT"] Server running at: http://localhost:3000
```

Log without tags:
```
2018-07-23T12:12:41.403Z [info] - server started
```

Would look cleaner to not show anything in case of no tags or an empty array

**Comment by felixheck** [comment]
Hi @5an1ty — thanks for using laabr and your issue.
I definitely agree with you in this case. 

The `-` should be more a placeholder for other tokens.
Even though `data.tags` is undefined and not an empty array – it's due to hapi, pino & hapi-pino – I would use an empty array in this case.

**Comment by felixheck** [comment]
Published as `v3.1.6` – thanks for contributing!
